lhotse/recipes/ami.py

```diff
@@ -265,7 +265,7 @@ class AmiSegmentAnnotation(NamedTuple):
 
 
 def parse_ami_annotations(
-    annotations_dir: Pathlike, normalize_text: bool = True
+    annotations_dir: Pathlike, normalize: str = "upper"
```

Comment:
why not make "kaldi" default here as well for consistency?

Reply:
Makes sense, I'll do it next time I raise a PR.